tests/integration-service/integration.go

```diff
@@ -19,7 +19,7 @@ import (
 )
 
 const (
-	containerImageSource   = "quay.io/redhat-appstudio-qe/busybox-loop:latest"
+	containerImageSource   = "quay.io/redhat-appstudio-qe/busybox-loop@sha256:f698f1f2cf641fe9176d2a277c9052d872f6b1c39e56248a1dd259b96281dda9"
```

Comment:
I think we can remove this line since the variable is not used any more in the file.

Reply:
Yes, thank you! Removed.